synapse/storage/schema/delta/53/add_user_type_to_users.sql

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Record the user_type of the user, initially this column is used to identify
```

Comment:
This is kinda stating the obvious and will be unhelpful if we ever add more user types. How about:

```
The type of the user: NULL for a regular user, or one of the constants in synapse.api.constants.UserTypes
```